LICENSE

```diff
@@ -279,6 +280,9 @@ This product includes code from Apache Spark.
 
 * dev/check-license script
 * vectorized reading of definition levels in BaseVectorizedParquetValuesReader.java
+* SQL grammar rules in IcebergSqlExtensions.g4
+* SQL parsing logic in IcebergSparkSqlExtensionsParser.java
+* parser post processing logic in IcebergSqlExtensionsPostProcessor.java
```

Comment:
I think it's quite a bit broader than this. How about "portions of the extensions parser"?

Reply:
Sounds good to me.

Reply:
Done.